docker-for-mac/faqs.md

```diff
@@ -10,7 +10,7 @@ title: Frequently asked questions (FAQ)
 
 ### How do I get the Stable or the Edge version of Docker Desktop?
 
-You can download the Stable version of Docker Desktop from [Docker Hub](https://hub.docker.com/?overlay=onboarding). To download the Edge version, see the [Edge release notes](/docker-for-mac/edge-release-notes/).
+You can download the Stable version of Docker Desktop from [Docker Hub](https://hub.docker.com/editions/community/docker-ce-desktop-mac). To download the Edge version, see the [Edge release notes](/docker-for-mac/edge-release-notes/).
```

Comment:
Should we provide the stable URL here - https://download.docker.com/win/stable/Docker%20Desktop%20Installer.exe?